Fabric.Authorization.Client/AuthorizationClient.cs

```diff
@@ -15,13 +16,29 @@ public AuthorizationClient(HttpClient client)
             this.client = client;
         }
 
-        public async Task<UserApiModel> GetPermissionsForCurrentUser()
+        public async Task<UserApiModel> GetPermissionsForCurrentUser(string accessToken)
         {
+            CheckAccessToken(accessToken);
+

```

Comment:
I missed this on the original code review, but does HttpRequestMessage need to be wrapping in a using? I think it implements IDisposable.